From 73c65a8962388a17c81292cf27e03295f493f63e Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Mon, 16 Apr 2018 15:26:54 +0300
Subject: [PATCH 6/6] Allow recovery write to sysfs_graphics

* I771556a254688b3e934d196717f755c783387704 broke it
  so recovery wasn't able to write to the brightness path

Change-Id: I1cc51176cabeafc3f4fb697fe0e434a943581a44
---
 qcom/private/recovery.te | 1 +
 1 file changed, 1 insertion(+)

diff --git a/qcom/private/recovery.te b/qcom/private/recovery.te
index 517a8aa..cab7350 100644
--- a/qcom/private/recovery.te
+++ b/qcom/private/recovery.te
@@ -2,4 +2,5 @@ recovery_only(`
 r_dir_file(recovery, sysfs_battery_supply)
 r_dir_file(recovery, sysfs_graphics)
 r_dir_file(recovery, sysfs_usb_supply)
+allow recovery sysfs_graphics:file write;
 ')
-- 
2.17.0

